docs/config.md

```diff
@@ -1192,6 +1193,15 @@ Default: `false`
 
 Type: `bool`
 
+### `Pubsub.SeenMessagesTTL`
+
+Configures the duration after which a previously seen Pubsub Message ID can be
+forgotten about.
```

Comment:
@smrz2001 mind elaborate that this is a setting for advanced users, and note what are sife-effects of setting it too high or too low?

Reply:
Just added some clarifications, @lidel.